### PR TITLE
Minor test improvements

### PR DESCRIPTION
**Pull request by hannes-ucsc**
Assorted fixes I stumbled upon during #980.

### CODE REVIEW

File: tests/test_doctests.py [hunk 1]
```diff
@@ -5,15 +5,20 @@
 Run selected doctests as a regular unit test.
 """
 
-import os
+import doctest
 import sys
+import unittest
+
```
Comment (kislyuk) [inline tests/test_doctests.py]: one newline too many here

Comment (hannes-ucsc) [reply tests/test_doctests.py]: PyCharm put that one there. I'm not sure why os is different from sys.

Will fix.

File: dss/api/search.py [hunk 2]
```diff
@@ -37,8 +37,10 @@ def post(json_request_body: dict,
 
     replica_enum = Replica[replica] if replica is not None else Replica.aws
 
-    logger.debug("Received posted query. Replica: %s Query: %s Per_page: %i Timeout: %s Scroll_id: %s",
-                 replica_enum.name, json.dumps(es_query, indent=4), per_page, _scroll_id)
+    if logger.isEnabledFor(logging.DEBUG):
+        logger.debug("Received POST for replica=%s, es_query=%s, per_page=%i, _scroll_id: %s",
+                     replica_enum.name, json.dumps(es_query, indent=4), per_page, _scroll_id)
```
Comment (kislyuk) [inline dss/api/search.py]: Wait, sorry, I missed this the first time around. Why is it necessary to gate `logger.debug` with `logger.isEnabledFor`? Shouldn't that logic happen automatically inside `logger`?

Comment (hannes-ucsc) [reply dss/api/search.py]: Yes, the interpolation is only performed if log level is debug or lower. However, `json.dumps` isn't exactly cheap. This is the recommended way to gate expensive computation of interpolation arguments: https://docs.python.org/3/howto/logging.html#optimization

Comment (kislyuk) [reply dss/api/search.py]: In general, I would recommend not worrying about the cost of json.dumps.

Comment (hannes-ucsc) [reply dss/api/search.py]: I'll take it out.

**Pull request merged**